src/Deserializers/LegacyItemDeserializer.php

```diff
@@ -101,7 +113,6 @@ private function getStatementList() {
 
 	/**
 	 * @param array $claimSerialization
-	 *
```

Comment:
Our current best practice is to always have an empty line between params and return.